app/controllers/auth_controller.rb

```diff
@@ -0,0 +1,21 @@
+class AuthController < ApplicationController
+  def new
+    render :login
+  end
+
+  def login
+    @user = User.find_by(username: params[:user][:username])
+    if @user && @user.password == params[:user][:password]
```

Comment:
extract this logic out and since it has to do with your data - it should live in the User model. maybe an `#authenticate` method?